swan-lake/learn/quick-tour.md

```diff
@@ -56,122 +63,71 @@ You get the following output.
 
 A Ballerina client is a component, which interacts with a network-accessible service. It aggregates one or more actions that can be executed on the network-accessible service and accepts configuration parameters related to the network-accessible service.
 
 There are two kinds of clients in Ballerina, inbound (or ingress) and outbound (or egress) clients. An outbound client object can be used to send messages to a network service.
 
-Having said that, let's see how you can use a Ballerina client to invoke the Hello World service.
+Create an outbound Ballerina client as a Ballerina program with a `main` function as follows to invoke the `hello` service.   
 
-First, you need to create the client with the relevant endpoint URL as follows. We will use a Ballerina program with a `main` function, which will perform the invocation.
-
-> **Note**: returning `error?` allows you to use the `check` keyword to avoid handling errors explicitly. This is only done to keep the code simple. However, in real production code, you may have to handle those errors explicitly.
-
-```ballerina
-http:Client helloClient = new("http://localhost:9090/hello");
-```
-
-As the next step, add the below code to do a `GET` request to the Hello World service.
-
-```ballerina
-http:Response helloResp = check helloClient->get("/sayHello");
-```
-
-The remote call would return an `http:Response` if successful, or an `error` on failure. If successful, attempt retrieving the payload as a `string` and print the payload.
-
-```ballerina
-io:println(check helloResp.getTextPayload());
-```
-
-The complete source code should look similar to the following:
+> **Note**: Returning `error?` from the `main` function allows you to use the `check` keyword to avoid handling errors explicitly. This is only done to keep the code simple. However, in real production code, you may have to handle those errors explicitly.
 
+***hello_client.bal***
 ```ballerina
 import ballerina/http;
 import ballerina/io;
 
 public function main() returns @tainted error? {
+
+    // Add the relevant endpoint URL to perform the invocation.
     http:Client helloClient = new("http://localhost:9090/hello");
-    http:Response helloResp = check helloClient->get("/sayHello");
+
+    // Perform a `GET` request to the `hello` service. If successful, 
+    // the remote call would return an `http:Response` or the payload 
+    // (if the `targetType` defaultable parameter is configured).
+    // Otherwise an `error` on failure.
+    http:Response helloResp = <http:Response> check helloClient->get("/sayHello");
+
+    // Retrieve the payload as a `string` and print it if the 
+    // response of the remote call is successful.
     io:println(check helloResp.getTextPayload());
 }
 ```
 
-Make sure the service is up and running.
-
-Now, you can run the `.bal` file containing the `main` function that invokes the service.
+### Invoking the 'hello' Service Using the 'helloClient'  
```

Comment:
Let's remove the "helloClient" terms as mentioned above.

Reply:
Done.